cmd/util/cmd/execution-state-extract/cmd.go

```diff
@@ -166,6 +171,16 @@ func run(*cobra.Command, []string) {
 		if _, err := os.Stat(flagInputPayloadFileName); os.IsNotExist(err) {
 			log.Fatal().Msgf("payload input file %s doesn't exist", flagInputPayloadFileName)
 		}
+
+		partialState, err := util.IsPayloadFilePartialState(flagInputPayloadFileName)
+		if err != nil {
+			log.Fatal().Err(err).Msgf("cannot get flag from payload input file %s", flagInputPayloadFileName)
+		}
+
+		// Check if payload file contains partial state and is allowed by --allow-partial-state-from-payload-file.
+		if !flagAllowPartialStateFromPayloads && partialState {
```

Comment:
can we include both `flagAllowPartialStateFromPayloads` and `partialState` into the log?